drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-testing/src/main/java/org/drools/workbench/screens/scenariosimulation/webapp/client/editor/ScenarioSimulationEditorKogitoTestingScreen.java

```diff
@@ -0,0 +1,172 @@
+import org.drools.workbench.screens.scenariosimulation.webapp.client.popup.ScenarioSimulationKogitoLoadingScesimPopupPresenter;
+import org.drools.workbench.screens.scenariosimulation.webapp.client.services.TestingVFSService;
+import org.gwtbootstrap3.client.ui.Popover;
+import org.jboss.errai.common.client.api.ErrorCallback;
+import org.kie.workbench.common.widgets.client.menu.FileMenuBuilder;
+import org.uberfire.backend.vfs.Path;
+import org.uberfire.backend.vfs.PathFactory;
+import org.uberfire.client.annotations.WorkbenchMenu;
+import org.uberfire.client.annotations.WorkbenchScreen;
+import org.uberfire.client.promise.Promises;
+import org.uberfire.lifecycle.OnStartup;
+import org.uberfire.mvp.Command;
+import org.uberfire.mvp.PlaceRequest;
+import org.uberfire.mvp.impl.DefaultPlaceRequest;
+import org.uberfire.workbench.model.menu.Menus;
+
+import static org.drools.workbench.screens.scenariosimulation.webapp.client.editor.ScenarioSimulationEditorKogitoTestingScreen.IDENTIFIER;
+
+@ApplicationScoped
+@WorkbenchScreen(identifier = IDENTIFIER)
+public class ScenarioSimulationEditorKogitoTestingScreen extends AbstractScenarioSimulationEditorKogitoScreen {
+
+    public static final String IDENTIFIER = "ScenarioSimulationEditorKogitoTestingScreen";
+    private static final String BASE_URI = "git://master@system/system/";
+    public static final String BASE_DMN_URI = BASE_URI + "stunner/diagrams/";
+    public static final String BASE_SCESIM_URI = BASE_URI + "scesim/";
+    public static final Path DMN_PATH = PathFactory.newPath("DMN", BASE_DMN_URI);
+    public static final Path SCESIM_PATH = PathFactory.newPath("SCESIM", BASE_SCESIM_URI);
+    private static final PlaceRequest SCENARIO_SIMULATION_KOGITO_TESTING_SCREEN_DEFAULT_REQUEST = new DefaultPlaceRequest(IDENTIFIER);
+
+    @Inject
+    protected ScenarioSimulationKogitoLoadingScesimPopupPresenter loadScesimPopupPresenter;
+
+    @Inject
+    protected FileUploadPopupPresenter fileUploadPopupPresenter;
+
+    @Inject
+    protected TestingVFSService testingVFSService;
+
+    @Inject
+    protected Promises promises;
+
+    @Override
+    public PlaceRequest getPlaceRequest() {
+        return SCENARIO_SIMULATION_KOGITO_TESTING_SCREEN_DEFAULT_REQUEST;
+    }
+
+    @OnStartup
+    @Override
+    public void onStartup(final PlaceRequest place) {
+        getItemsByPath(SCESIM_PATH);
+        getItemsByPath(DMN_PATH);
+        addTestingMenus(scenarioSimulationEditorKogitoWrapper.getFileMenuBuilder());
+        super.onStartup(place);
+    }
+
+    protected void getItemsByPath(Path path) {
+        testingVFSService.getItemsByPath(path, response -> {
+            // do nothing
+        }, (message, throwable) -> {
+            if (throwable instanceof org.uberfire.java.nio.file.NotDirectoryException) {
+                testingVFSService.createDirectory(path);
+                return true;
```

Comment:
I don't like this piece of code because I don't see the reason to create a path if doesn't exist but considering that it is for testing purpose I'm fine to keep it :)